### PR TITLE
Add "Install themes" button on updates page

### PR DESCRIPTION
**Pull request by FlusherDock1**
Very oftenly in our community people asking question:"How do i install theme from marketplace?" 
And to answer that question we should say "Just press Install plugins button!"
That's not very intuitive for beginners. This PR adds one button that leads right to themes installation page.

### CODE REVIEW

**Comment by LukeTowers** [comment]
@FlusherDock1 could you add a before / after screenshot?

File: modules/system/controllers/updates/_list_toolbar.htm [hunk 1]
```diff
@@ -11,6 +11,11 @@
         class="btn btn-success oc-icon-plus">
         <?= e(trans('system::lang.plugins.install')) ?>
     </a>
+    <a
+        href="<?= Backend::url('system/updates/install/themes') ?>"
+        class="btn btn-success oc-icon-plus">
+        <?= e(trans('system::lang.themes.install')) ?>
+    </a>
```
Comment (LukeTowers) [inline modules/system/controllers/updates/_list_toolbar.htm:18]: Just move this before the install plugins button and it should be good to go

**Comment by FlusherDock1** [comment]
> @FlusherDock1 could you add a before / after screenshot?

Before:
image

After:
image

File: modules/system/controllers/updates/_list_toolbar.htm (hunk 1 above)
Comment (FlusherDock1) [reply modules/system/controllers/updates/_list_toolbar.htm]: Done.

**Comment by LukeTowers** [comment]
Thanks @FlusherDock1!

**Pull request merged**